src/app.js

```diff
@@ -1 +1,10 @@
 'use strict';
+
+const minimist = require('minimist');
+
+const lastName = minimist(process.argv.slice(2)).lastname;
+const age = minimist(process.argv.slice(2)).age;
+const name = minimist(process.argv.slice(2)).name;
```

Comment:
you can just destructure it
```suggestion
const { name, lastname, age } = minimist(process.argv.slice(2));
```